gulp/tasks/build.js

```diff
@@ -7,7 +7,6 @@ gulp.task( 'build',
     'styles',
     'scripts',
     'images',
-    'copy',
-    'usage'
+    'copy'
```

Comment:
Trivial: You could leave the comma on this line to reduce the diff to one line, here.